### ISSUE TITLE
Lightbulb won't show up if diagnostic or selection is at the beginning of a file

### ISSUE THREAD

**Issue by heejaechang** [microsoft/pylance-release#5156]
1. create a python file
2. put "os" in the file. make sure that's only text in the file and `end of file` is at the end of `os` not the next line

expected: lightbulb shows up.
actual: no lightbulb

select the text `os`

expected: lightbulb shows up
actual: no lightbulb

...

if you put `end of file` to the next line (hit enter after `os`), everything work as expected.

**Comment by StellaHuang95** [comment]
I can't repro this with v2023.11.14
![image](https://github.com/microsoft/pylance-release/assets/100439259/7dac0788-0696-4bec-824a-841fa5df539c)

**Comment by StellaHuang95** [comment]
@heejaechang Does this still repro for you? If not, I am going to close it.

**Comment by heejaechang** [comment]
let me try again

**Comment by heejaechang** [comment]
dup of https://github.com/microsoft/vscode/issues/200792
